cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/workflow/ProgramWorkflowAction.java

```diff
@@ -60,22 +61,22 @@ public void configure() {
 
   @Override
   public void run() {
+    String prettyProgramType = programType != null ? ProgramType.valueOf(programType.name()).getPrettyName() : "";
```

Comment:
Sure. I am just using the existing approach, this just being the change for logs. Will check further.